filesystem/src/main/java/org/frankframework/filesystem/IMailFileSystem.java

```diff
@@ -15,40 +15,59 @@
 */
 package org.frankframework.filesystem;
 
-import org.xml.sax.SAXException;
+import java.io.IOException;
+import java.util.Iterator;
+import java.util.Map;
 
 import org.frankframework.stream.Message;
 import org.frankframework.xml.SaxElementBuilder;
+import org.xml.sax.SAXException;
 
-public interface IMailFileSystem<M,A> extends IWithAttachments<M,A> {
+public interface IMailFileSystem<M,A> extends IBasicFileSystem<M> {
 
-	public final String MAIL_MESSAGE_ID = "Message-ID";
-	public final String RETURN_PATH_HEADER="Return-Path";
+	String MAIL_MESSAGE_ID = "Message-ID";
+	String RETURN_PATH_HEADER="Return-Path";
 
 	/*
 	 * IMailFileSystems should make sure the additionalProperties contain also the following keys:
 	 */
-	public final String TO_RECEPIENTS_KEY = "to";
-	public final String CC_RECEPIENTS_KEY = "cc";
-	public final String BCC_RECEPIENTS_KEY = "bcc";
-	public final String FROM_ADDRESS_KEY = "from";     // originator of the message
-	public final String SENDER_ADDRESS_KEY = "sender"; // identifies who submitted the messages, probably on behalf of the 'From'
-	public final String REPLY_TO_RECEPIENTS_KEY = "replyTo";
-	public final String DATETIME_SENT_KEY = "DateTimeSent";         // as Date, or in XML format: yyyy-MM-dd'T'HH:mm:ss.SSSZ
-	public final String DATETIME_RECEIVED_KEY = "DateTimeReceived";
+	String TO_RECEPIENTS_KEY = "to";
```

Comment:
Perhaps you change the typo here while you're working in this file to 'RECIPIENTS' (same for the two below)?